packages/cadl-ranch-specs/http/models/special-characters/main.cadl

```diff
@@ -0,0 +1,33 @@
+import "@azure-tools/cadl-azure-core";
+import "@azure-tools/cadl-ranch-expect";
+
+using Cadl.Rest;
+using Cadl.Http;
+using Azure.Core.Foundations;
+
+@serviceTitle("SpecialCharacterClient")
+@serviceVersion("1.0.0")
+@server("http://localhost:3000", "TestServer endpoint")
+@doc("Illustrates models with special characters or keyword in property name")
+@route("/special-chars")
+namespace Model.SpecialCharacter;
+
+@doc("Base type with a discriminator containing dot.")
```

Comment:
I don't think `special-characters` should be tied to models. We can have a separate issue to add `special-characters` tests directly under `http`. This way we can test special characters for mdoels, operations, etc

Reply:
removed `special-characters` from this PR